README.md

```diff
@@ -1,31 +1,31 @@
 # JAVASCRIPT ADVANCED DISCUSSION GROUP
-Join us for a round table of analyzing, critical thinking and creating with JavaScript! Think of this as a way to test everything you've learned so far and use creativity as a tool to deepen your fundamentals.
+Join us for an hour of of analyzing, critical thinking and creating with JavaScript! Think of this as a way to test everything you've learned so far and use creativity as a tool to deepen your fundamentals.
 ## Pre-requisites:
-- JavaScript Track: JS130, Up to Testing
+- JavaScript Track: JS130
 - Ruby Track: JS225
-- Understand Bloom's [Taxonomy Launch School Presentation](https://www.reddit.com/r/launchschool/comments/kvf6pv/blooms_taxonomy_x_launch_school/), [General introduction](https://tophat.com/blog/blooms-taxonomy/) and [List of Questions](https://www.eoas.ubc.ca/research/cwsei/resources/Blooms-verbs-stems.htm).
+- Check out Bloom's Taxonomy with one of these: [Launch School Presentation](https://www.reddit.com/r/launchschool/comments/kvf6pv/blooms_taxonomy_x_launch_school/), [General introduction](https://tophat.com/blog/blooms-taxonomy/) and [List of Questions](https://www.eoas.ubc.ca/research/cwsei/resources/Blooms-verbs-stems.htm).
 ## Goals for the Group
-- Practice problem solving collaboratively & intentionally
-- Group programming
-- Improve communication and teamwork
-- Focus on levels 4,5 and 6 of the Taxonomy
+- Practice intentional problem solving collaboratively 
+- Practice technical communication and precision
+- Practice levels 4,5 and 6 of the Taxonomy
 ## Guidelines for Participants
-If you feel comfortable with a question, make space for others first, then help them fill their gaps
-This is a space for everyone to contribute, participate, take risks, experiment,  to be open-minded to learning and LS Code of Conduct and Community Guidelines. 
+- If you feel comfortable with a question, make space for others first, then help them fill their gaps. 
+- This is a space for everyone to contribute, participate, take risks and experiment. Be open-minded, willing to learn and respect the LS Code of Conduct and Community Guidelines. 
+- Everyone should lead the session at least once.
+- Weekly rotation of leader.
```

Comment:
Maybe better: 'We encourage everyone to lead a session at least once'  or 'Everyone can be a leader for this group, as long as you have an interesting problem to solve or discussion topic to share'. 'should' may sound a bit harsh. What do you think?

Reply:
Yes, I think the tone is better the way you write it.